content/cloud/noops-sql-db.md

```diff
@@ -34,24 +35,21 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
 * Your NoOps SQL Database can hold up 1 GB of data
 
 *Service Limitations*
-* The `--sqlite` flag not supported on `spin cloud deploy`
+* The `--sqlite` flag is not supported on `spin cloud deploy`
 * Fermyon **Cloud** only supports databases named "default" at this time
 
-## How To:
+## How To
 
-### Prepare a Database
+Firstly, if you have not already done so, you will need to [create your first Spin application](https://developer.fermyon.com/spin/quickstart#create-your-first-application). Our blog posts [Announcing Fermyon Cloud’s NoOps SQL Database](https://www.fermyon.com/blog/announcing-noops-sql-db) and [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) each have an example of how to create a NoOps SQLite Storage application. To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest]((https://developer.fermyon.com/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components)) (the spin.toml file). For example:
 
-You can use the `spin cloud sqlite execute` command to check if a table needs to be created, and if so create one: 
-
-```bash
-$ spin cloud sqlite execute inspirational-pig "CREATE TABLE IF NOT EXISTS todos (id INTEGER PRIMARY KEY AUTOINCREMENT,description TEXT NOT NULL,due_date DATE,starred BOOLEAN DEFAULT 0,is_completed BOOLEAN DEFAULT 0)"
+```
+[component]
+sqlite_databases = ["default"]
 ```
 
->> If you need to look up the name of your database, use the `spin cloud sqlite list` command
-
-### Manage the Database Lifecycle
+### Database Access From Many Components
 
-A database is created when you deploy a Spin application that references a "default" SQLite database in its component manifest. Multiple components in the same Spin application can be given access to the default database like so:
+A database is created when you build a Spin application referencing a "default" SQLite database in the component section of an application's manifest. Multiple components in the same Spin application can be given access to the same `default` database like so:
```

Comment:
"deploy" was correct - `spin build` does _not_ create a database.  (Also I prefer "that references" because it can only apply to "application", whereas "referencing" could modify "build".)

```suggestion
A database is created when you deploy a Spin application that references a "default" SQLite database in the component section of an application's manifest. Multiple components in the same Spin application can be given access to the same `default` database like so:
```

Reply:
Ah, yes. Apologies. I had it in my head that `build` created the database environment due to my following workflow example (below). I just now realized that the `--sqlite` I am using is an option of an `up` subcommand. So technically, the database is indeed being created and populated as part of a deployment, not a `build`. Thanks for pointing this out.

```
$ spin new http-rust
Enter a name for your new application: test_sqlite
Description: Test SQLite
HTTP base: /
HTTP path: /...
$ cd test_sqlite 
# add the sqlite_databases = ["default"] to manifest file
$ spin build
$ spin up --sqlite "CREATE TABLE IF NOT EXISTS character (id INTEGER PRIMARY KEY AUTOINCREMENT, firstname TEXT NOT NULL, lastname TEXT NOT NULL)" --sqlite "INSERT INTO character VALUES (NULL,'Lois','Lane')"
$ spin cloud sqlite list
Databases (1)
vivacious-mangosteen (default)
```

Reply:
I will commit this as part of my next push; thanks for the wording.

Reply:
Yes, but regardless, this is _NOT_ creating the cloud database.  `spin cloud sqlite list` is listing databases that exist in the cloud (which are created on `deploy`.  `spin up` creates (if needed) a _local_ database, by default in the application's `.spin` directory.  Your mangosteen database is one that happened to already exist in your cloud account.